README.md

```diff
@@ -113,3 +115,17 @@ export default {
 
 </script>
 ```
+
+## refreshToken
+
+This will shortcut getting a refresh token from Google, this should be placed in your App.vue on the created page and run on a timer of 45min
+```html
+<script>
+  name: 'App'
+
+  created () {
+  // NOTE: this throws errors when the user is signed out....
+  window.setInterval(this.$refreshToken(), 2.7e+6)
```

Comment:
@hkd987 what's the purpose of the interval & it's huge (`2.7e+6`) delay?

Reply:
That is 45min. That is the recommended time that google ask you to refresh. The token is dead after one hour, and google says not to refresh more than every 45min.